### DISCUSSION TITLE
How to schedule a relay for this one to work properly

### DISCUSSION THREAD

**Discussion by villaverdefm** [AzuraCast/AzuraCast#6043] [Q&A]
Hi, friends.
I like connecting my station to other ones and it's difficult for me to have this without rebooting the AutoDJ set.

When scheduling shows from other radios I have to do this for this connection to be stablished. Is there any way to do that and this show to be programmed and broadcasted properly without doing this?

I hope to make myself understood, Thanks in advance.

**Comment by ChrisDuffley** [comment]
@villaverdefm:
Yes, this can be done by setting the remote URL playlist to "Interrupt track to play at scheduled time" under Advanced.

@BusterNeece: Is there a way this can be implemented where 1 interrupted playlist can interrupt another, or does Liquidsoap not have an option for this at this time, since those take priority over uninterrupted ones anyway?

**Reply by villaverdefm** [reply]
Hi. I'm afraid this option mentioned by Chris doesn't work because what I want to schedule are URLs instead of M3U playlists. And doing this this option always fails.

**Reply by ChrisDuffley** [reply]
@villaverdefm, this does not fail for me, I've seen it not work sometimes but that's not always consistent. It does indeed work with URL's, I do it every week to relay programming from another station on a private one. I don't know why it doesn't work for you.

What version of AC are you on? I've seen the Stable have issues with this, but I'm on Rolling and haven't had any problems.

**Reply by villaverdefm** [reply]
I'm using the last stable version. But I could change to the rolling release and check if it works there.

**Reply by ChrisDuffley** [reply]
@villaverdefm, I've seen it work on a station just now I'm helping manage that's also on Stable and it works fine. You may have to restart Liquidsoap for it to work right but it does work properly pretty consistently here.

**Reply by villaverdefm** [reply]
How to restart Liquidsoap without stopping my station from time to time?
